### ISSUE TITLE
Can not view video

### ISSUE THREAD

**Issue by saurabhPV** [GetStream/stream-chat-swift#83]
## What did you do?

I uploaded the small video from camera, on 1.5.4 it shows reaction view and on 1.5.5 it crashes

## What did you expect to happen?

View video on taping link.

## What happened instead?
Crashes when I tap video.
``` 
Thread 1: EXC_BAD_INSTRUCTION (code=EXC_I386_INVOP, subcode=0x0)
```

## GetStream Environment
**GetStream Chat version:** 1.5.5
**iOS version:**  iOS 12
**Swift version:**  5
**Xcode version:** 11.3.1
**Device:** 

## Additional context

Logs from Xcode console
```🐴 ⏱ Response decoded: 1.765 +0.009
stream-chat-swift/Sources/UI/Web View/WebViewController.swift: 155: 19: Fatal error: Use of unimplemented initializer 'init(nibName:bundle:)' for class 'StreamChat.WebViewNavigationController'
2020-01-30 17:46:03.070103+0530 ChatExample[10610:205186] 
stream-chat-swift/Sources/UI/Web View/WebViewController.swift: 155: 19: Fatal error: Use of unimplemented initializer 'init(nibName:bundle:)' for class 'StreamChat.WebViewNavigationController'
```

**Comment by b-onc** [comment]
Hello @saurabhPV ,

Are you using storyboards? How did you setup the chat?

**Comment by saurabhPV** [comment]
For 1.5.5 I am using an example app. With CocoaPods.
For 1.5.4 I am using an example but I installed pods from CocoaPods rather than from local directory.
I have uploaded a video from 1.5.4 but then I saw you have fixed issue ( shows  reaction view on tap) in 1.5.5 so I cloned the repo and installed pods

**Comment by b-onc** [comment]
@saurabhPV You mean Example app in this repo, right? Not the demo/tutorial app? Can you add more steps? We are not able to reproduce the issue using Example app in this repo.

**Comment by saurabhPV** [comment]
I was able to reproduce it again

1. git clone https://github.com/GetStream/stream-chat-swift.git
2. In  Example/Cocoapods ran pod install 
3. In login screen used key obtained from dashboard, entered one number in user id , then name (This is the same user and conversation I created in 1.5.4, 1 or 2 days ago)
4. Clicked on message that I had sent when I was testing 1.5.4

Also, I was able to grab the link of video and sent that link as message and in 1.5.4 when I click on that message with url I can view video.
